src/Http/HttpHelper.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Castor\Http;
+
+use Castor\Helper\PathHelper;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/** @internal */
+class HttpHelper
```

Comment:
Can you find a better name without the *helper* suffix?

Btw, I'm not sure the proxy part to the http client is really useful since we have de DIC now.

If you remove it, the naming will become clearer